specifyweb/frontend/js_src/lib/components/Interactions/PrepReturnRow.tsx

```diff
@@ -175,3 +175,63 @@ export function PrepReturnRow({
     </>
   );
 }
+
+export function handleReturnChanged({
```

Comment:
handle* naming convention is usually used for functions that react to change by updating the sate, and those functions usually don't return anything. in your case, the [function is pure](https://en.wikipedia.org/wiki/Pure_function), and it does return a value

perhaps a better name would be `updateReturnCount` and `updateResolveCount`